schema/deploy/shipping/views.sql

```diff
@@ -844,4 +844,95 @@ comment on view shipping.hcov19_observation_v1 is
   'Custom view of hCoV-19 samples with presence-absence results and best available encounter data';
 
 
+create or replace view shipping.scan_return_results_v1 as
+
+    with status_codes as (
+        -- Create status codes as required by Lab Med's results portal
+        with x(status_code, status) as (
+            values
+            ('DET', 'positive result, colored red + bold'),
+            ('INCON', 'inconclusive, colored orange; treated as positive with respect to information given'),
+            ('NDET', 'not detected/negative, colored blue'),
+            ('pending-other', null),
+            ('pending-lab', 'We are working on it!'),
+            ('cancelled', null),
+            ('notReceived', 'This is our code we made up')
+        )
+        select * from x
+    ),
+
+    results as (
+        select distinct on (barcode, lineage)
+            sample_id,
+            barcode as qrcode,
+            encountered as collect_ts,
+            case
+                when sample_id is null then 'notReceived'  -- todo new code
+                when sample_id is not null and present is null then 'pending-lab'
+                when target.identifier is not null and present is null then 'INCON'
+                when present is true then 'DET'
+                when present is false then 'NDET'
+            end as status_code,
+            target.identifier,  -- todo debug
+            lineage,  -- todo debug
+            present  -- todo debug
+
+        from warehouse.identifier
+        join warehouse.identifier_set using (identifier_set_id)
+        left join warehouse.sample on uuid::text = sample.collection_identifier
+        left join warehouse.encounter using (encounter_id)
+        left join warehouse.presence_absence using (sample_id)
+        left join warehouse.target using (target_id)
+        left join warehouse.organism using (organism_id)
+
+    where identifier_set.name in('collections-scan', 'collections-swab&send') -- todo
+        and target.control is distinct from true
+        and (lineage is null or
+            lineage <@ '{"Human_coronavirus.2019"}'::ltree[])
```

Comment:
I think we might want to add a date cutoff? We don't want to report on earlier samples.

Reply:
I think it's fine if we just stick to `collections-scan`.

Reply:
Yeah, made that comment when I thought we were also returning SFS results through the same website.